site/pages/search-results-fr.hbs

```diff
@@ -0,0 +1,240 @@
+			<ul>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<!--Note: if labels go past 1 row, then option a) cut off at end of row, option b) do not show any labels beyond the first row. Text-overflow of the cut off label should be with an ellipsis. Expand/collapse to show all is not a requirement for Jan 2018 MVP-->
+				<!--Note 2: First label should always be the label of the context the search is in (if contextualised). Not necessarily order of tags in page metadata.-->
+			<ul>
+			<ol class="breadcrumb">
+				<li class="[link]">[domain name of search result]</li>
+				<li class="[link]">[parent breadcrumb of page]</li>
+				<!--Note: this always has 2 list items. Does not include other breadcrumb items.-->
+			<ol>
+			<p><time datetime="[dateModified of search result]">[DD MMM. YYYY]</time> - [description/highlight of search result. Search terms in <strong>]</p>
+		</section>
+		<section>
+			<h3><a href="[link]">[Search result page title (not including '- Canada.ca' or other institutional suffix)]</a></h3>
+			<ul>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<!--Note: if labels go past 1 row, then option a) cut off at end of row, option b) do not show any labels beyond the first row. Text-overflow of the cut off label should be with an ellipsis. Expand/collapse to show all is not a requirement for Jan 2018 MVP-->
+				<!--Note 2: First label should always be the label of the context the search is in (if contextualised). Not necessarily order of tags in page metadata.-->
+			<ul>
+			<ol class="breadcrumb">
+				<li class="[link]">[domain name of search result]</li>
+				<li class="[link]">[parent breadcrumb of page]</li>
+				<!--Note: this always has 2 list items. Does not include other breadcrumb items.-->
+			<ol>
+			<p><time datetime="[dateModified of search result]">[DD MMM. YYYY]</time> - [description/highlight of search result. Search terms in <strong>]</p>
+		</section>
+		<section>
+			<h3><a href="[link]">[Search result page title (not including '- Canada.ca' or other institutional suffix)]</a></h3>
+			<ul>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<!--Note: if labels go past 1 row, then option a) cut off at end of row, option b) do not show any labels beyond the first row. Text-overflow of the cut off label should be with an ellipsis. Expand/collapse to show all is not a requirement for Jan 2018 MVP-->
+				<!--Note 2: First label should always be the label of the context the search is in (if contextualised). Not necessarily order of tags in page metadata.-->
+			<ul>
+			<ol class="breadcrumb">
+				<li class="[link]">[domain name of search result]</li>
```

Comment:
What is the use of the CSS class ```class="[link]"```. Is it needed?

>  This should be shown as the domain name of the page with a chevron “>” followed by the parent breadcrumb of the page. 

Is the intention to have each individual item to be clickable and pointing to different link, like a breadcrumb? If so, the markup would look like ```<li><a href="[link]">[breadcrumb name]</a></li>```

> When this is not possible, the breadcrumb can be replaced by the page URL.

Do you have a example of that? It might require a different markup, like a paragraph or a cite element.

Reply:
Individual item shouldn't be clickable. I'll remove class there and add the example URL markup